infrastructure/main.tf

```diff
@@ -88,5 +88,5 @@ module "postgresql_flexible" {
     }
   ]
 
-  pgsql_version = "14"
+  pgsql_version = "15"
```

Comment:
What was the reason behind choosing 15 versus 16 for the staging platform?
Also has someone been able to backup the staging database / schema - just incase it needs to be restored as this is the first time doing a major version bump?
FYI @jamesmcstravick mentioned something about this earlier - so please ask him to approve when ready to see what happens here.

Reply:
Platops Recommendation to move to 15. Discussed with @jamesmcstravick and he's happy to go ahead. Looking to do it now to reduce impact since its start of sprint.